### PR TITLE
{vis}[intel/2015a] motif 2.3.4 with libX11 1.6.3 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

File: easybuild/easyconfigs/m/motif/motif-2.3.4-intel-2015a-libX11-1.6.3.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    ('libXext', '1.3.3'),
+    ('libXdmcp', '1.1.2'),
+    ('libpng', '1.6.16'),
+    ('xbitmaps', '1.1.1'),
+    ('freetype', '2.5.5'),
+    ('libjpeg-turbo', '1.4.0'),
+    ('bzip2', '1.0.6'),
+]
+builddependencies = [
+    ('libtool', '2.4.5', '', ('GCC', '4.9.2')),
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/motif/motif-2.3.4-intel-2015a-libX11-1.6.3.eb]: `autotools`?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 17 out of 18 (5 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/dabde57a028c0a7cea2d for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 21 out of 22 (5 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/ee3a0af10e1d3e6f1aad for a full test report.

**Comment by boegel** [comment]
@wpoely86: I've added lowopt: True (i.e. use -O1), which fixes the segfault on my end, please retest?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/67f0911c45632c16c46a for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 22 out of 22 (5 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/6f48d97df07b67f4fb4d for a full test report.

**Comment by boegel** [comment]
Hooray! Thanks for reviewing/testing @wpoely86!

**Pull request merged**